docs/application/web/guides/device/feedback.md

```diff
@@ -4,6 +4,9 @@ You can handle feedback patterns (media or vibration) that can be played as a re
 
 This feature is supported in mobile and wearable applications only.
 
+To guarantee the running of the application on a device which supports feedback vibration feature, since 5.0 declare the following feature requirement in the config file:
+- http://tizen.org/feature/feedback.vibration
```

Comment:
As you can see in other files(for example /web/guides/app-management/service-app.md), 
we have "Prerequisites" section, which describe the things should be prepared before start.

How about adding "Prerequisites" section and describe "http://tizen.org/feature/feedback.vibration" at there?

Another example, /web/guides/connectivity/iotcon.md
![image](https://user-images.githubusercontent.com/9958374/43873395-701b01d6-9bc1-11e8-92bc-a21108b411ea.png)